src/main/java/org/wso2/siddhi/plugins/idea/runconfig/RunConfigurationKind.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.siddhi.plugins.idea.runconfig;
+
+/**
+ * Defines kinds of run configurations.
+ */
+public enum RunConfigurationKind {
+    MAIN
```

Comment:
Since Siddhi does not have multiple run kinds, why didn't this was removed?

Reply:
Had the doubt about this whether we need more types in future. Will fix after clarifying this with the team.

Reply:
Fixed in commit 3aa87a6